deps/build.jl

```diff
@@ -1,7 +1,16 @@
-# If BLAS was compiled with MKL, we want to use MKL for FFTs as well. Thus
-# we have to do this little dance to get around having to use BinDeps for
-# a library that's already linked to Julia.
-if Base.BLAS.vendor() === :mkl
+# If BLAS was compiled with MKL and the user wants MKL-based FFTs, we'll oblige.
+# In that case, we have to do this little dance to get around having to use BinDeps
+# for a library that's already linked to Julia.
+settings = joinpath(@__DIR__, "..", ".build_settings")
+if isfile(settings)
+    provider = readchomp(settings)
+elseif haskey(ENV, "JULIA_FFTW_PROVIDER")
+    provider = ENV["JULIA_FFTW_PROVIDER"]
+    open(f -> println(f, provider), settings, "w")
+else
+    provider = ""
+end
+if provider == "MKL" && Base.BLAS.vendor() === :mkl
```

Comment:
Shouldn't it be an error if `provider == "MKL"` and MKL is not available?

Reply:
Yeah that's probably better